lib/browser/desktop-capturer.js

```diff
@@ -28,45 +28,47 @@ ipcMain.on('ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', function (event, cap
 
   // If the WebContents is destroyed before receiving result, just remove the
   // reference from requestsQueue to make the module not send the result to it.
-  event.sender.once('destroyed', function () {
+  event.sender.once('destroyed', () => {
     request.webContents = null
+    return
```

Comment:
Why an explicit `return` is needed here?

Reply:
whoops, removing